frontend/src/hooks/api/actions/useChangeRequestApi/useChangeRequestApi.ts

```diff
@@ -75,9 +75,9 @@ export const useChangeRequestApi = () => {
         trackEvent('change_request', {
             props: {
                 eventType: payload.state,
-                previousState,
+                ...(previousState ? { previousState } : {}),
```

Comment:
There is no previous state for new drafts, so let's leave that field out in that case.